src/ProjectTask.php

```diff
@@ -1106,9 +1133,51 @@ public static function showFor($item)
 
         $iterator = $DB->request($criteria);
         if (count($criteria)) {
+            $massive_action_form_id = 'mass' . str_replace('\\', '', static::class) . $rand;
+            if ($canedit) {
+                Html::openMassiveActionsForm($massive_action_form_id);
+                if ($deleted) {
+                    $massiveactionparams = [
+                        'num_displayed' => min($_SESSION['glpilist_limit'], count($criteria)),
+                        'is_deleted' => $deleted,
+                        'specific_actions' => [
+                            'restore' => _x('button', 'Restore'),
+                            'purge' => _x('button', 'Delete permanently')
+                        ]
+                    ];
+                } else {
+                    $massiveactionparams = [
+                        'num_displayed' => min($_SESSION['glpilist_limit'], count($criteria)),
+                        'is_deleted' => $deleted,
+                        'specific_actions' => [
+                            'update' => _x('button', 'Update'),
+                            'clone' => _x('button', 'Clone'),
+                            'delete' => _x('button', 'Put in trashbin')
+                        ]
+                    ];
+                }
+                Html::showMassiveActions($massiveactionparams);
+
+                echo "<label class='form-check form-switch btn btn-sm btn-ghost-danger me-0 me-sm-1 px-1 mb-0 flex-column-reverse flex-sm-row'
+                    data-bs-toggle='tooltip' data-bs-placement='bottom' title='" . __('Show the trashbin') . "'>";
+                echo "<input type='checkbox' class='form-check-input ms-0 me-1 mt-0' role='button'
+                    name='is_deleted' value='1' autocomplete='off'
+                    onclick=\"document.location.search = document.location.search.replaceAll(/&is_deleted=(1|0)/g, '') + '&is_deleted=' + (this.checked ? 1 : 0)\"
+                    " . ($deleted ? "checked" : "") . " />";
+                echo "<span class='form-check-label mb-1 mb-sm-0'>";
+                echo "<i class='ti fa-lg ti-trash'></i>";
+                echo "</span>";
+                echo "</label>";
```

Comment:
In other sublists, we used to display both deleted and non deleted items at the same time, and apply a "red" background on the deleted items.

You should keep this logic, it would prevent to have to introduce a new `is_deleted` switch and have to pass its value through the different methods.

For instance, on the Appliances tab of a computer.
![image](https://github.com/glpi-project/glpi/assets/33253653/f0197200-c2bd-4cae-b7d8-7e75bed18548)


```suggestion
```